js/script.js

```diff
@@ -0,0 +1,22 @@
+window.onload = () => {
+  const mobileBtn = document.querySelector('.mob-icon');
+  const mobileBtnClose = document.querySelector('.close-btn');
+  const mobileBody = document.querySelector('.mobile-body');
+  const navMain = document.querySelector('.main-nav');
+  // const mainLogo = document.querySelector(".main-logo");
```

Comment:
- [ ] Please remove this commented line of code, as well as the one at `line 11`, because leaving commented code in your codebase is not a good practice and prevents you from having a clean code base.👌